assets/js/googlesitekit/data/utils.js

```diff
@@ -212,6 +212,55 @@ export const collectName = ( ...args ) => {
 	return names.shift();
 };
 
+/**
+ * Combines multiple stores.
+ *
+ * @since n.e.x.t
+ *
+ * @param {...Object} args A list of objects, each a store containing one or more of the following keys: INITIAL_STATE, actions, controls, reducer, resolvers, selectors
+ * @return {Object} The combined store.
+ */
+export const combineStores = ( ...args ) => {
+	const combined = {};
```

Comment:
Minor change here but I think it would be a little cleaner to return the new object here rather than build a variable
```suggestion
	const combined = {};
	return {
		INITIAL_STATE: collectState( /*  */ ),
		actions: collectActions( /*  */ ),
		controls: collectControls( /*  */ ),
		selectors: collectSelectors( /*  */ ),
		resolvers: collectResolvers( /*  */ ),
		reducer: collectReducers( /*  */ ),
	};
```
You can keep the comments for each if you'd like (I think they're a bit more relevant in this context).

Reply:
Yeah, reading through this it'd be a lot neater to nix the `combined = {};` variable and return the newly-built-in-place object.

Reply:
Yeah I was considering that, but didn't since I needed the `INITIAL_STATE` to be used for building the `reducer` as well. But I can just build that initial state first and for the rest I'll just return the newly-built-in-place object. The comments seem like overkill when the object's being built in place, since it's a little clearer what's happening.